[16.0][OU-ADD] fetchmail: nothing to do

huguesdk opened the pull request:
`fetchmail` was merged into `mail` without model or data changes, and it is already present in the list of merged modules in `apriori.py`.

legalsylvain commented:
/ocabot migration fetchmail

legalsylvain commented on docsource/modules150-160.rst:
```diff
@@ -138,7 +138,7 @@ Module coverage 15.0 -> 16.0
 | event_sms                                       |                      |No DB layout changes.                            |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| fetchmail                                 |                      |                                                 |
+| |del| fetchmail                                 | Nothing to do        | Merged into mail.                               |
```
```suggestion
| |del| fetchmail                                 | Done        | Merged into mail.                               |
```

(as something is done in the apriori.py file).

huguesdk replied:
thanks for the review @legalsylvain! fixed and updated git comment.

legalsylvain reviewed (commented):
minor remarks. Otherwise, LGTM.

huguesdk committed:
[OU-ADD] set fetchmail as done

fetchmail was merged into mail without model or data changes, and it is
already present in the list of merged modules in apriori.py.

legalsylvain reviewed (approved):
/ocabot merge nobump

legalsylvain commented:
@huguesdk Could you rebase ?

huguesdk commented:
sure, but why is it needed, as there are no conflicts? the ci failed during `pre-commit` with the error “premature close”, as if the process was somehow aborted.

legalsylvain commented:
let's retry.

/ocabot merge nobump
